sageintacctsdk/apis/ar_invoices.py

```diff
@@ -0,0 +1,29 @@
+"""
+Sage Intacct AR Invoice
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class ARInvoices(ApiBase):
+    """Class for AR Invoice APIs."""
+    def __init__(self):
+        ApiBase.__init__(self, dimension='ARINVOICE')
```

Comment:
Since `ARINVOICE` uses legacy api for creating, we should initialise class something like this `ApiBase.__init__(self, dimension='ARINVOICE', post_legacy_method='create_invoice')`